src/list.js

```diff
@@ -0,0 +1,17 @@
+import Task from './task';
+
+export default class List {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || List.initialize();
+  }
+
+  static initialize() {
+    const task1 = new Task('Eat', 0);
+    const task2 = new Task('Rest', 1);
+    const task3 = new Task('Sleep', 2);
+    return [task1, task2, task3];
+  }
+}
+
+const list = new List();
+console.log(list.tasks);
```

Comment:
- [ ] Kindly try to delete the console.log statements. It is not a good practice to push it with code.